packages/waggle/src/storage/storage.ts

```diff
@@ -181,7 +183,6 @@ export class Storage {
       await this.channels.load({ fetchEntryTimeout: 2000 })
       this.io.loadPublicChannels({
         communityId: this.communityId,
-        // @ts-expect-error
         channels: this.channels.all
```

Comment:
We have a problem with this - lerna bootstrap says it's unused, but CI will fail without this

Reply:
Damn, that's bad :/